packages/modules/pgr/src/components/Modal.js

```diff
@@ -35,9 +38,13 @@ const Modal = (props) => {
   //   }, [file]);
 
   useEffect(async () => {
+    setError(null);
     if (file) {
-      const response = await Digit.UploadServices.Filestorage(file);
-      setUploadedFile(response.data.files[0].fileStoreId);
+      if (file.size > 5242880) {
+        setError("Maximum upload size exceeded");
```

Comment:
@mahendra-egov add localization text

Reply:
I didnt added localization bcz
1. We dont have localization key of it as of now
2. We will get error message from server instead of hard coded localization key